README.md

```diff
@@ -22,8 +22,17 @@ As you embark on these exercises, you're set to learn key practices in modern ap
 
 By the end of these exercises, you'll have a solid foundation in deploying and managing Kubernetes applications using Argo CD.
 
+> Hint to advanced topics that are beneficial to expert users. Like:
+> * Develop Argo CD App Structure & test features in isolation
+> * Developing & validating [Plugins](https://argo-cd.readthedocs.io/en/stable/operator-manual/config-management-plugins/)
+> * [UI Customization](https://argo-cd.readthedocs.io/en/stable/operator-manual/ui-customization/) etc. without impacting the (usually) just one live Argo CD instance.
+> * Notifications
+> * all other advanced features
+
 ## Getting Started
 
+> Link to https://codespaces.new/kutespaces/argocd
```

Comment:
people get a little confused when they never used a codespace. Whats your preferred way to intruct them step by step?

1. go to profile->settings->default editor to vs code, then open link https://codespaces.new/kutespaces/argocd
2. go to top right corner, open codespace, wait for it to load, go to top left corner, press open in codespace
3. open https://codespaces.new/kutespaces/argocd, wait for it to set up, then go to top left corner and open in vscode

3 sounds best now that I think about it.

Reply:
Opening the Codespace in the browser first comes with a risk. If the user forgets to open in VSCode, it almost works fine except for the ingress. I wish they added the dropdown to choose where to open it to the initial step.
I'm for (1) or (3) with no strong preference.

btw: As soon as the Codespace is created, you can specify it in the URL: https://github.com/codespaces/silver-space-fishstick-j6j7vwwjcqrwp?editor=vscode